airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/SchemalessTypesToJson.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data
+
+import io.airbyte.cdk.load.data.json.toJson
+import io.airbyte.cdk.load.message.DestinationRecord
+import io.airbyte.cdk.load.util.serializeToString
+
+class SchemalessTypesToJson : AirbyteSchemaIdentityMapper {
```

Comment:
this file + its test also looks like a dup from the previous PR? (SchemalessTypesToJsonString.kt)

Reply:
rebase artifact